components/grade-form.js

```diff
@@ -0,0 +1,24 @@
+class GradeForm{
+    constructor(formElement){
+        this.formElement = formElement;
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.formElement.addEventListener('submit', this.handleSubmit);
+    }
+    onSubmit(createGrade){
+        this.createGrade = createGrade;
+
+    }
+    handleSubmit(event){
+        event.preventDefault();
+        console.log("hi");
```

Comment:
Please remove this `console.log`, now that you've verified this functionality works.